CRM/Civicase/Form/Report/Case/CaseWithActivityPivot.php

```diff
@@ -120,7 +120,7 @@ public function __construct() {
     ];
 
     $caseColumns = $this->getColumns('Case', ['fields' => FALSE]);
-    $caseClientContactColumns = $this->getColumns('Contact', ['prefix_label' => 'Case Client - ', 'group_title' => 'Case Client Contact']);
+    $caseClientContactColumns = $this->getColumns('Contact', ['prefix_label' => 'Case Client - ', 'group_title' => 'Contacts']);
```

Comment:
@tunbola  Should label and title have localisation tag?

Reply:
No need. Already handled here: https://github.com/compucorp/uk.co.compucorp.civicase/blob/95fc9218ce996cff32edefceda3498d128530ca8/CRM/Civicase/Form/Report/ColumnDefinitionTrait.php#L284